src/migrations/00043-add-belongs-to-family-dt.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "device type" 
+ADD COLUMN IF NOT EXISTS "belongs to-device family" TEXT NULL;
```

Comment:
I'm assuming this is from a previous version because right now this looks like it should be a foreign key

Reply:
@Page- yup it was my silly mistake, fixed